README.md

```diff
@@ -1 +1,224 @@
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Bookstore <a name="about-project"></a>
+
+>In this react project I will lay the foundations for my Bookstore website. I will create the React part of the app. I will also set up routing using React
+
+**Bookstore app**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+  - HTML
+- CSS
+- [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+- [React]((https://github.com/microverseinc/curriculum-react-redux/blob/main/Bookstore/lessons/what_is_react.md))
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+    <li>React</li>
+  </ul>
+</details>
+<details>
+  <summary>Bundler</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Github</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Click on Categories and Display a Status**
+- **Click on Books and display a list of Books**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+#### 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add a book**
+- [ ] **Add an author**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/ritikarawat220/Bookstore.git
+  cd Bookstore
+```
+
+### Install
+
+Install this project with:
+
+```yarn
+```
+```sh
+  npm install
+```
+
+```yarn
+```
+```sh
+   npm install --save-dev @testing-library/react
+```
+
+
+```yarn
+```
+```sh
+   npm install --save-dev react-test-renderer
+```
+
+### Usage
+
+To run the project in a development server, execute the following command:
+
+```sh
+  npm start
+```
+
+### Testing
+
+To run the tests in a development server, execute the following command:
+
+```yarn
+```
+```sh
+  npm test
+```
+
+To to build for the production, execute the following command:
+
+```yarn
+```
+```sh
+  npm run build
+```
+
+### Deployment
+
+You can deploy this project using [Github Pages]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ritika Rawat**
+
+- GitHub: [ritikarawat220](https://github.com/ritikarawat220)
+- Twitter: [@ritikarawat22](https://twitter.com/Ritikarawat22)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rawatritika/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
```

Comment:
- [ ] The link to the issue page is broken kindly make sure to update it so it redirects to the issues page instead